### ISSUE TITLE
Bug 54111260: [React Native Gallery -> Print]: 'Print' and 'Cancel' buttons are getting truncated after applying text scaling to 200%.

### ISSUE THREAD

**Issue by Yajur-Grover** [microsoft/react-native-gallery#502]
Build Details:
OS Version: Dev (OS Build 27695.1000)
App: React Native Gallery 
App Version: 1.0.18.0
React Native Windows Version: 0.75.0

AT: Text Scaling  
Resolution: 1280*768

Pre-Requisite :  
Text Scaling (Settings->Accessibility->Text size-> Drag the slider to 200% ->Apply))

Repro-Steps:
Launch the React Native Gallery app.
Navigate to ‘All samples’ button present in left navigation pane and invoke it.
Navigate to ‘Print’ button present under “Media” group and invoke it.
Navigate to 'Print PDF' button and invoke it.
Now observe 'Print' and 'cancel' buttons.
Actual Result:
'Print' and 'Cancel' buttons are getting truncated after applying text scaling to 200%.

Expected Result:
'Print' and 'Cancel' buttons should be visible clearly to the user without any data loss and truncation.

User impact:

Users with low vision will not be able to consume the content of the button if it is not visible in Text scaling mode.

Link to ADO bug: https://dev.azure.com/microsoft/OS/_workitems/edit/54111260

**Comment by Yajur-Grover** [comment]
The issue here is on the Print window which is being called by accessing the RNPrint API, part of the `react-native-print` repo. @chrisglein Not sure if this is under our scope since we don't directly own the module, but it does say that there is Windows support, so if this is on us, I will look into making a change on the react-native-print repo.

**Comment by Yajur-Grover** [comment]
Closing as this has been redirected to the Print team, and is not an issue with RNW directly.
